### ISSUE TITLE
Category Anchor Highlighting does not correspond to actual position in scroller.

### ISSUE THREAD

**Issue by georgehrke** [serebrov/emoji-mart-vue#77]
There is an issue with the Emoji Mart Category Anchors. They do not update properly, as you scroll the emoji-picker. I took a small screencast to demonstrate the problem:

![Kapture 2020-07-17 at 11 05 52](https://user-images.githubusercontent.com/1250540/87769296-b7b63580-c81d-11ea-904d-f151fcdb2123.gif)

The "People & Body" category becomes highlighted while I'm still at the very beginning of "Smileys & Emotion", etc.
The screencast was taken on https://serebrov.github.io/emoji-mart-vue/, but is also reproducible in my local project. 

It was taken with Chromium 84 on macOS 10.15. However, this bug is also reproducible in all other tested browser / os combinations. It is not limited to the "native" emoji set, but also to all others.

**Comment by serebrov** [comment]
I see the issue and this is only reproduced in `Picker` component that uses virtual scroller. There is no such issue in `StaticPicker`.

I am pushing the fix, although I am not 100% sure if it will work in all cases. Before, I used `endIndex-1` expression to highlight the active category (where endIndex is passed in by the virtual scroller). Now, it looks like this is off by 1 and I've changed it to `endIndex-2`.

In general, virtual scroller seems to case issues (#57, #50) and that was the reason I also added `StaticPicker`. Maybe it is worth to switch the default component (have `Picker` to be the "static picker" and add a `VirtualPicker` as alternative for performance tuning).
